### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by farishkash** [Pcole1/prj-rev-bwfs-dasmoto#3]
Excellent, So when I review this project I look for two things, use of divs and no repetition of css.

Divs are by nature design to divide each section of your web page into manageable containers.
So in the beginning some do not realize why they should use them. I always mention this because as you progress through the program you will see that you will put everything into a div..

Your css was excellent, I mention the class naming convention more of a heads up as I do not know your goals with the ready program when it comes to job hunting.

As a challenge take a look at bootstrap.  See if you can add it to this site to make it mobile friendly.  It will also heavily reinforce the concept of using divs holding sections of content.

**Comment by Pcole1** [comment]
You're right about the divs.  I didn't understand them at first but they're definitely critical and when I look at code of pages I see a lot of them.  And I can see how you learn how to use them properly and more creatively as you move along.

I thought CSS would be more difficult because it's not as linear when coding as HTML, but it was actually pretty simple in this project.

What is bootstrap again? 

Thanks for the review and critiques!
